addons/sys_intercom/stringtable.xml

```diff
@@ -165,6 +165,7 @@
             <Polish>Nie można się połączyć do interkomu. Limit połączeń został osiągnięty.</Polish>
             <Italian>Impossibile connettersi all'intercom. Numero massimo di connessioni raggiunto.</Italian>
             <French>Connexion impossible à l'interphone. Nombre maximal de connexions atteint.</French>
+            <German>Maximale Anzahl an Verbindungen in Bordfunk erreicht. Verbindung kann nicht hergestellt werden!</German>
```

Comment:
Suggestion -> `Maximale Anzahl an Verbindungen in Bordfunk erreicht. Verbindung kann nicht hergestellt werden.` (No exclamation mark)

Reply:
no exclemation mark and typo